packages/mobile/src/components/core/AnimatedButton.tsx

```diff
@@ -21,23 +21,20 @@ import { usePrevious } from 'react-use'
 import { GestureResponderHandler } from 'app/types/gesture'
 import { Theme, useThemeVariant } from 'app/utils/theme'
 
-export type BaseAnimatedButtonProps = {
+type IconJSON = any
```

Comment:
oh i guess we already had `any` but does `JSON` type work?

Reply:
`JSON` doesn't work but I was able to pull the type off the Lottie props!